dependencies.yaml

```diff
@@ -137,9 +162,8 @@ dependencies:
               - cudatoolkit=11.8
```

Comment:
Please drop the version pinnings from this section. Edit all the lines like `cudatoolkit=11.8` so it's just `cudatoolkit`, which will be constrained by `cuda-version=11.8`.

Reply:
sure! done in https://github.com/rapidsai/wholegraph/pull/115/commits/99472f639539ef34adc17146595ffa61041546ec